osu.Game.Modes.Osu/Objects/OsuHitObjectParser.cs

```diff
@@ -63,23 +110,15 @@ public override HitObject Parse(string text)
 
                         string[] temp = pointsplit[i].Split(':');
                         Vector2 v = new Vector2(
-                            (int)Convert.ToDouble(temp[0], CultureInfo.InvariantCulture),
-                            (int)Convert.ToDouble(temp[1], CultureInfo.InvariantCulture)
+                            (int)Convert.ToDouble(temp[0], NumberFormatInfo.InvariantInfo),
+                            (int)Convert.ToDouble(temp[1], NumberFormatInfo.InvariantInfo)
                         );

```

Comment:
I would prefer this to be `(int)Convert.ToDouble(temp[1], NumberFormatInfo.InvariantInfo));`.